src/modules/aws_ec2/entity/instance_block_device_mapping.ts

```diff
@@ -27,13 +27,16 @@ export class InstanceBlockDeviceMapping {
    * Device name to associate this volume to the instance
    */
   @Column()
+  @cloudId
```

Comment:
Why add these `@cloudId` decorators if you are overriding the `entityId` and `generateId` methods?

Reply:
i answered to Alejandro previously as well, if i do not add any @cloudId the tooling complains... so it didn't complain before because there was a leftover for that in `region` field. If i remove all @cloudId code fails. So i prefered to put the reasonable ones. But please suggest any other approach that would work, i would like to get rid of those cloudId as well.